src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
+spring.datasource.url=jdbc:mysql://localhost:3306/mysql
+spring.datasource.username=root
+spring.datasource.password=root
```

Comment:
Change to `application.yml`
Is a good practice add an application name using `spring.application.name` property

Reply:
```
spring:
   datasource:
       url:
       username:
       password:
       
```